mmf/modules/encoders.py

```diff
@@ -125,6 +125,15 @@ def __init__(self, config: Config, *args, **kwargs):
         self.lc.bias.data.copy_(torch.from_numpy(bias))
         self.out_dim = out_dim
 
+        def state_dict_key_override(state_dict, prefix, *args, **kwargs):
```

Comment:
move this outside of the class. I am wondering if there is a better way to achieve this hooking via registry.

Reply:
I have a feeling that directly overridding the `_load_from_state_dict` method might be more sustainable. See how pytorch does this: https://github.com/pytorch/pytorch/blob/8aaca4b46ad5856219a72c7164f67ab7211f70ab/torch/nn/quantized/modules/linear.py#L206

Reply:
Also, check better way of doing this through class: https://github.com/pytorch/pytorch/blob/6dc763df3098485f158eeed2b9621cffaa229669/torch/nn/utils/spectral_norm.py#L149

Reply:
I changed it to using the `_load_from_state_dict to` override the state key